tests/exampleRepoTests/nodeJS-v2.smoke.test.ts

```diff
@@ -0,0 +1,16 @@
+const swaggerJson = require('../../test-repos/nodeJS-v2/swagger/swagger.js');
+
+describe('nodeJS-v2', () => {
+  beforeAll(async () => {
+    console.log('result', swaggerJson);
+  });
+
+  it('general Info', async () => {
+    expect(swaggerJson.info.title).toEqual('nodeJS-v2');
+    const paths = Object.keys(swaggerJson.paths);
+    expect(paths.length).toEqual(3);
+
+    const definitions = Object.keys(swaggerJson.definitions);
+    expect(definitions.length).toEqual(4);
```

Comment:
We should probably make a TODO or file an Issue to add actual parameter validation here instead of simply checking the lengths, right?